src/styles/components/_component-card.scss

```diff
@@ -21,23 +22,55 @@
       margin: auto;
     }
 
+    // the following breakpoints are where certain illustrations try to break out of their containers
+    @media only screen and (min-width: 475px) {
+      height: 30rem; // 480px/16px
+    }
+
+    @media only screen and (min-width: 690px) {
+      height: 40rem; // 640px/16px
+    }
+
+    @media only screen and (min-width: $breakpoint-tablet) {
+      height: 100%;
+    }
+
     &:hover img {
       scale: 102%;
     }
 
     &__header {
-      font-weight: 700;
-      font-size: 1.25rem; // 20px
+      font-family: "Oxygen Bold", sans-serif;
+      z-index: 1;
+      font-size: 1.125rem; // 18px/16
       line-height: 4.6667rem; // 75px
-      letter-spacing: 2%;
-      text-align: center;
-      display: block;
-      border-top: 0.25rem solid map.get($colors, "black");
+      letter-spacing: 0.02em;
+      border-top: 0.25rem solid var(--foreground-color);
+      background-color: var(--background-color);
+      position: relative;
```

Comment:
Same thing here: Change the `border-top` to black/background-color; change the 'background-color` to white/foreground-color.

Reply:
I just switched these! :)